toucan_connectors/aircall/aircall_connector.py

```diff
@@ -194,3 +247,18 @@ def _retrieve_data(self, data_source: AircallDataSource) -> pd.DataFrame:
             return build_df(
                 dataset, [empty_df, pd.DataFrame(team_data), pd.DataFrame(variable_data)]
             )
+
+    def get_status(self) -> ConnectorStatus:
+        """
+        Test the Aircall connexion.
+        """
+        try:
+            access_token = self.get_access_token()
+        except Exception:
+            return ConnectorStatus(status=False, error='Credentials are missing')
+        if not access_token:
+            return ConnectorStatus(status=False, error='Credentials are missing')
```

Comment:
If everything is fine, I think we should return `ConnectorStatus(status=True)`, to indicate everything is OK, shouldn't we?

Reply:
Yes, I added a return of ConnectorStatus with status True and wrote the relevant test.